roles/kubernetes/node/templates/kubelet.service.j2

```diff
@@ -11,6 +11,9 @@ Wants={{ container_manager }}.service
 [Service]
 User=root
 EnvironmentFile=-{{ kube_config_dir }}/kubelet.env
+{% if not kubelet_rotate_server_certificates %}
```

Comment:
It would be better to handle this like this:

`{% if kubelet_rotate_server_certificates | default(false) %}`

unless the policy is to have everything defined in defaults

Reply:
`kubelet_rotate_server_certificates` already exists to enable RotateKubeletServerCertificate
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation
We can't enable it by default because k8s doesn't approve kubelet CSR by default for security reasons, so you need some extra components

When it's disabled, kubelet generate a self-signed cert that is never regenerated. This is not really a problem because it's not checked at all.
The idea of this PR was to delete this self-signed cert so it's regenerated, but not sure it really provide any value